meta-mbl-distro/recipes-support/swupdate/files/defconfig

```diff
@@ -0,0 +1,98 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Swupdate Configuration
+#
+CONFIG_HAVE_DOT_CONFIG=y
+
+#
+# Swupdate Settings
+#
+
+#
+# General Configuration
+#
+CONFIG_SCRIPTS=y
+CONFIG_HW_COMPATIBILITY=y
+CONFIG_HW_COMPATIBILITY_FILE="/etc/hwrevision"
```

Comment:
Does `/etc/hwrevision` get created at build time? If so, we might be able to use the hw compatibility feature, but I'd suggest that the file belongs in our factory config partition.

Reply:
I'm testing with this config removed now. I had to manually create `/etc/hwrevision` in my previous test.

Reply:
Seems to work with the HW_COMPATIBILITY switched off in my simple test. I can't find much in the `swupdate` documentation about this config. Do you think we should leave it switched off?

Reply:
I can only see it used in one place in the code (parser/parser.c). It looks like if we switch it off it will just ignore hardware compatibility checks in sw-description files.

I think we should turn it off until we decide we want to use it.